modules/nw-cluster-mtu-change.adoc

```diff
@@ -0,0 +1,309 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-network-mtu.adoc
+
+[id="nw-cluster-mtu-change_{context}"]
+= Changing the cluster MTU
+
+As a cluster administrator, you can change the maximum transmission unit (MTU) for your cluster. The migration is disruptive and nodes in your cluster might be temporarily unavailable as the MTU update rolls out.
+
+.Prerequisites
```

Comment:
I have no idea if it would be safe to assume that all of the cluster nodes are running with the same network configuration. I not, It might be worth it to add a prerequisite on that respect.

Reply:
@jcaamano, can you elaborate a bit on _same network configuration_? Is this the same MTU everywhere or the same network card or the same routes or something else? Thanks!

Reply:
@jcaamano, I moved this into the procedure:

> If your hardware MTU is specified in a NetworkManager connection configuration, complete the following steps. This approach is the default for {product-title} if you do not explicitly specify your network configuration with DHCP, a kernel command line, or some other method. Your cluster nodes must all use the same underlying network configuration for the following procedure to work unmodified.

Reply:
@jcaamano, so I think in theory, someone could setup additional MachineConfigPools and have different MTU configurations, but I wanted to avoid covering every possible scenario for this. If someone from QE has ideas for DHCP or other approaches, I can add them, but I can't derive them locally as I lack the expertise and hardware setup to do so. I'm trying to cover what I hope are the most common cases.